config/rootfs/debos/scripts/install-firmware.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
```

Comment:
To avoid silent failures:
```suggestion

set -e
```